LiveSplit.AutoSplitters.xml

```diff
@@ -13351,6 +13351,16 @@
         <Type>Script</Type>
         <Description>Happenlance Developer AutoSplitter</Description>
         <Website>https://github.com/pmttavara/HappenlanceAutoSplitter</Website>
+    </AutoSplitter>
+    <AutoSplitter>
+        <Games>
+            <Game>Inscryption</Game>
+        </Games>
+        <URLs>
+            <URL>https://gist.github.com/Andersama/2160d1cfc1c95341607b728ba39d538f</URL>
```

Comment:
@Andersama You need to provide the RAW url to a file on a GitHub or Gitlab repository.
Gists aren't supported. Just change it in a second commit and I'll be happy to merge your request :)

Reply:
ah, that would make sense